### ISSUE TITLE
Python docstring examples for execute_script() and execute_async_script() are not helpful

### ISSUE THREAD

**Issue by paulie4** [SeleniumHQ/selenium#4393]
## Expected Behavior -
The "Usage" examples would be working examples.

## Actual Behavior -
The "Usage" examples are not working examples.

## Steps to reproduce -
I think the docstrings from https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/remote/webdriver.py#L512, which uses `driver.execute_script('document.title')` as the example usage for both execute_script() and execute_async_script(), are used to create the [readthedocs.io Python docs](http://selenium-python.readthedocs.io/api.html#selenium.webdriver.remote.webdriver.WebDriver.execute_script), so those docstrings should be fixed to have examples like this:
```python
result = driver.execute_script('return document.title')
```
and
```python
result = driver.execute_async_script("""
var done = arguments[0];
asyncFuncWithCallback(function (retVal) {
    done(retVal);
});
""")
```

**Comment by majora2007** [comment]
This is really needed. I had a problem understanding how to work with promises and execute_script until I found this post.

**Comment by p0deje** [comment]
I guess this is an easy one for a pull request.

**Comment by KingAkeem** [comment]
Hello, I'm new to the open source community. I've used Selenium for a few projects before and wanted to contribute. Are you saying you'd just like the documentation to actually contain some JavaScript in the docs to better explain what the execute_script/execute_async_script do?

**Comment by lmtierney** [comment]
@KingAkeem There's already an open PR for these changes in #4590 but there hasn't been any action in quite some time. If you look in that PR you should find some guidance

**Comment by KingAkeem** [comment]
Thank you very much

**Comment by lmtierney** [comment]
Fixed in 01131b2
